server/item/item.go

```diff
@@ -192,6 +194,18 @@ func (d defaultFood) ConsumeDuration() time.Duration {
 	return DefaultConsumeDuration
 }
 
+// DisplayName returns the display name of the item as shown in game in the language passed.
```

Comment:
Might be worth documenting that `DisplayName` panics if an unknown (unregistered) item is passed.